Barbara Kosior/35. Search Insert Position.cpp

```diff
@@ -0,0 +1,33 @@
+class Solution {
+public:
+    int searchInsert(vector<int>& nums, int target) {
+         
+        int i=0;
+        int b = nums.size()-1;
+
+        while (i<b){
+            int srednia = (i+b)/2;
+            if (nums[srednia] == target)
+            {
+                return srednia;
+            }
+            else if (nums[srednia] > target)
+            {
+                b = srednia -1;
+            }
+            else
+            {
+                i = srednia +1;
+            }
+        }
+        if (nums[i]>=target)
```

Comment:
You can also use a new feature in c++
return (nums[i]>=target) ? i : i+1;

Reply:
Okay, do I have to make a task to use this function?